react/README.md

```diff
@@ -1,14 +1,21 @@
 # React Tech Guides
+At [Wolox](https://wolox.co), we use our own version of [create-react-app](https://github.com/Wolox/wolox-react-scripts#creating-an-app) to create our React projects. Check it out to find a detailed guide to start with your project.
 
-## Technologies
+<!-- In construction -->
+<!-- ## Technologies
 
 - [React](./react/)
 
 - [React Native](./react-native/)
 
-- [React in Rails](./react-in-rails/)
+- [React in Rails](./react-in-rails/) -->
 
-## Our React Cross-Platform Open Source Libraries
+
+## Open Source
```

Comment:
I'd let `Our React Cross-Platform Open Source Libraries` because originally we put in this readme all cross platfom `things` , so I'd move this libs to `React` section. Think that also react native people will be working soon in this tech guides